training/model_management/src/run_model_preprocess.py

```diff
@@ -82,19 +94,37 @@ def _validate_transformers_args(args):
         preprocess_args.update(download_details.get("tags", {}))
         preprocess_args.update(download_details.get("properties", {}))
     preprocess_args["task"] = task_name if task_name else preprocess_args.get("task")
-    preprocess_args["model_id"] = model_id if model_id else preprocess_args.get("model_id")
+    preprocess_args["model_id"] = (
+        model_id if model_id else preprocess_args.get("model_id")
+    )
 
     print(preprocess_args)
 
     if mlflow_flavor == ModelFlavor.TRANSFORMERS.value:
         _validate_transformers_args(preprocess_args)
 
-    run_preprocess(mlflow_flavor, model_path, mlflow_model_output_dir, **preprocess_args)
+    run_preprocess(
+        mlflow_flavor, model_path, mlflow_model_output_dir, **preprocess_args
+    )
 
     # Copy license file in input model_path
     if license_file_path:
         shutil.copy(license_file_path, mlflow_model_output_dir)
 
+    # Validation to check if license file exist or not
```

Comment:
can you share why license file validation is required? And why not use `license_file_path`?